arangograph/projects.md

```diff
@@ -87,15 +87,42 @@ You can create an allowlist as part of a project.
 
 ## How to manage certificates
 
-Certificates are utilized for encrypted remote administration.
+Certificates are utilized for encrypted remote administration. The communication
+with and between the servers of an ArangoGraph deployment is encrypted using the
+TLS protocol.
 
-Certificates that do have the __Use well known certificate__ option enabled do
+Each ArangoGraph deployment is accessible on two different port numbers:
+- default port (8529)
+- high port (18529)
+
+The distinction between these port numbers is in the certificate used for the
+TLS connection.
+
+On the default port (8529), a **well known X509 certificate** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} is being used. This
+certificate has a lifetime of 5 years and is rotated automatically. It is
+recommended to use Well known certificates, as this eases the access of a
+deployment in your browser.
```

Comment:
+ Unless you switch off the 'Use well known certificate' setting in the certificate generation, in that case both the default and high port will serve the same self-signed certificate